### PR TITLE
First attempt at logging plugin for quartermaster

### PR DESCRIPTION
**Pull request by rattboi**
Fixes bonnyci/projman#156

Signed-off-by: Bradon Kanyid <bradon@kanyid.org>

### CODE REVIEW

**Review by jasontclark — COMMENTED** [review]
👍 LGTM, but let's get one more review before we merge.

File: err-logging/logging.py
Comment (eggshell) [inline err-logging/logging.py]: Would it maybe make more sense to have this be an absolute path?

Comment (eggshell) [inline err-logging/logging.py]: Would it maybe make more sense to have this be an absolute path?

**Review by eggshell — COMMENTED** [review]
looks pretty good, just left one question inline.

File: err-logging/logging.py
Comment (rattboi) [reply err-logging/logging.py]: Hmm, I'd like the plugin to be agnostic to where errbot is keeping it's stuff. However, I think I may need to update the systemd files on quartermaster side to make sure it executes from quartermaster's homedir, so that relative paths make sense.

Comment (eggshell) [reply err-logging/logging.py]: That seems like a good solution. Thanks!

**Review by eggshell — APPROVED** [review]

File: err-logging/logging.py
Comment (rattboi) [reply err-logging/logging.py]: This is dealt with in https://github.com/BonnyCI/hoist/pull/251

**Pull request merged**
